### DISCUSSION TITLE
Slider滑块实现反向坐标轴功能

### DISCUSSION THREAD

**Discussion by JIAGUI125** [element-plus/element-plus#15204] [General]
建议Slider滑块实现反向坐标轴功能,比如有些场景在垂直坐标系中，需要向下拉动滑块，同时向下滚动列表，目前垂直方向滑块只能向上滑动，即便是角度转换180度也无效，滑块还是只能向上滑动，无法满足需求。

**Comment by tuskermanshu** [comment]
+1   目前存在这样的需求

**Comment by forever-z-133** [comment]
听上去只需要封装一下，`min="-100" max="0"` 然后出入参处理下正负号即可
